### PR TITLE
Fix/category bugs

### PR DESCRIPTION
**Pull request by arf-hamza**
fix bugs for add category and edit category !

### CODE REVIEW

File: tp-3/src/app/[locale]/categories/page.tsx [hunk 1]
```diff
@@ -6,25 +6,18 @@ import {
   postApiCategory,
   putApiCategory,
   deleteApiCategory,
+  APICategory,
 } from "@/api/category.api";
 
-import { Box, Button, Grid } from "@mui/material";
-
```
Comment (glitterbabi) [inline tp-3/src/app/[locale]/categories/page.tsx:12]: it's a good idea to remove things we don't use : )

File: tp-3/src/components/organisms/category-list/category-list.tsx [hunk 2]
```diff
@@ -2,25 +2,23 @@
 
 import React, { useState, useEffect } from "react";
 import {
-    APICategory,
```
Comment (glitterbabi) [inline tp-3/src/components/organisms/category-list/category-list.tsx:5]: aren't those the same? that is so odd

File: tp-3/src/languages/fr.json [hunk 3]
```diff
@@ -32,7 +32,7 @@
     "modifyTitle": "Modifier la Catégorie",
     "saveButton": "Sauvegarder",
     "cancelButton": "Annuler",
-    "modifyMessage": "La catégorie a été modifiée avec succès",
+    "modifyMessage": "La catégorie a été enregistrer avec succès",
```
Comment (glitterbabi) [inline tp-3/src/languages/fr.json:35]: the change is actually wrong here! on devrait écrire 'a été enregistrée', car c'est la catégorie (féminin) qui a été enregistrée : )

**Review by glitterbabi — CHANGES_REQUESTED** [review]
the only change required is the last comment, just to make sure we don't forget hehe. Great job as always!

File: tp-3/src/languages/fr.json (hunk 3 above)
Comment (arf-hamza) [reply tp-3/src/languages/fr.json:35]: okay good

File: tp-3/src/app/[locale]/categories/page.tsx (hunk 1 above)
Comment (arf-hamza) [reply tp-3/src/app/[locale]/categories/page.tsx:12]: yes it's removed : )

**Review by glitterbabi — APPROVED** [review]
great! thanks a lot hehe

**Pull request merged**